native/.flowconfig

```diff
@@ -0,0 +1,45 @@
+[ignore]
```

Comment:
we don't use flow (yet...), so this file can be deleted ..?

Reply:
I'll try it and see what happens, should be fine.

Reply:
Done